alerter.cpp

```diff
@@ -8,7 +8,10 @@ int networkAlertStub(float celcius) {
     // Return 200 for ok
     // Return 500 for not-ok
     // stub always succeeds and returns 200
-    return 200;
+    if (celcius < 200)

```

Comment:
Do not fix the production code. Intention is to define tests/asserts such that it expresses the behavior. The failure of  tests is an indication of bug in code.